modules/profile/variables.tf

```diff
@@ -0,0 +1,40 @@
+########################################################################################################################
+# Input variables
+########################################################################################################################
+
+variable "instance_id" {
+  type        = string
+  description = "The ID of the SCC instance in a particular region."
+}
+
+variable "profile_name" {
+  type        = string
+  description = "The profile name. Constraints: The maximum length is `64` characters. The minimum length is `2` characters. The value must match regular expression `/[A-Za-z0-9]+/`."
+}
+
+variable "profile_description" {
+  type        = string
+  description = "The profile description. Constraints: The maximum length is `256` characters. The minimum length is `2` characters. The value must match regular expression `/[A-Za-z0-9]+/`."
+}
+
+variable "control_libraries" {
```

Comment:
This is not what I would expect as input to create a custom profile. From the profiles perspective it just needs needs a list of controls to add to the profile. They can be predefined (aka come from an existing control library), or custom (one you created yourself).